react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class SwitchButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      button: 'on'
+    };
+    console.log(this.state.button)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    if (this.state.button === 'on') {this.setState ({button: 'off'})}
+    else {this.setState ({button: 'on'})}
+  }
+
+  render() {
+    if (this.state.button === 'on') {
+        return (
+            <div className = {`lights ${this.state.button}`}>
+                <button className={`switch-button ${this.state.button}`}>
+                </button> 
+                <button onClick={this.handleClick} className={`button-position ${this.state.button}`}></button>   
+                ON    
+            </div>
+        )        
+    }
+    else {
+        return (
+            <div className = {`lights ${this.state.button}`}>
+                <button className={`switch-button ${this.state.button}`}>
+                </button> 
+                <button onClick={this.handleClick} className={`button-position ${this.state.button}`}></button>    
+                OFF   
+            </div>
+        ) 
+    }
```

Comment:
Both of these are almost identical the only difference is the text so a better way to handle this would to be do it without the if statement and use a ternary to set the text:

```JSX
return (
  <div className = {`lights ${this.state.button}`}>
    <button className={`switch-button ${this.state.button}`}></button> 
    <button onClick={this.handleClick} className={`button-position ${this.state.button}`}></button> 
    { this.state.button ? 'ON' : 'OFF' }    
  </div>
);
```

Reply:
All done.